modules/govuk_jenkins/templates/jobs/deploy_terraform_govuk_aws.yaml.erb

```diff
@@ -0,0 +1,66 @@
+---
+- scm:
+    name: govuk-aws_Deploy_Terraform_Govuk_AWS
+    scm:
+        - git:
+            url: git@github.com:alphagov/govuk-aws.git
+            branches:
+                - master
+- job:
+    name: Deploy_Terraform_Project
+    display-name: Deploy_Terraform_Project
+    project-type: freestyle
+    description: |
+        Deploy Terraform from https://github.com/alphagov/govuk-aws
+
+    properties:
+        - github:
+            url: https://github.com/alphagov/govuk-aws/
+    scm:
+        - govuk-aws_Deploy_Terraform_Govuk_AWS
+    wrappers:
+        - ansicolor:
+            colormap: xterm
+        - build-name:
+          name: '${ENV,var=ENVIRONMENT} ${ENV,var="STACKNAME"} ${ENV,var="PROJECT"} ${ENV,var="COMMAND"}'
+    builders:
+        - shell: |
+            ./jenkins.sh
+
+    parameters:
+        - string:
+            name: AWS_ACCESS_KEY_ID
+            description: Your AWS access key ID
+            default: false
+        - password:
+            name: AWS_SECRET_ACCESS_KEY
+            description: Your AWS secret access key
+            default: false
+        - password:
+            name: AWS_SESSION_TOKEN
+            description: The AWS session token
+            default: false
+        - choice:
+            name: COMMAND
+            description: Choose whether to plan (default), apply or destroy
+            choices:
+                - plan
+                - apply
+                - destroy
+        - choice:
+            name: ENVIRONMENT
+            description: Which environment to deploy to
+            choices:
+                - integration
+                - staging
+                - production
+        - choice:
+            name: STACKNAME
+            description: Choose which stack to deploy into
+            choices:
+                - blue
+                - green
+        - string:
+            name: PROJECT
+            description: Name of the project you wish to deploy
```

Comment:
Can we make this a `choice` and include all of the `infra-` and `app-` projects programatically somehow, or is that way too much work? Or we could make it a list? But then that's _another_ place to update when we add a new one? Or is people typing still OK?

Reply:
I've [been there before](https://github.com/alphagov/govuk-puppet/commit/64a8df8eab)! I'm not opposed to it, but in my experience it was a pain to keep updated and didn't bring a huge amount of benefit. I'm not entirely opposed to it if people would prefer a dropdown for usability.

Reply:
Ah! OK then. It is another place to update which would get in the way a bit, but then so will mis-typing a project and waiting for Jenkins to return a failure. But people can complain abut this when/if they start using it, I suppose. Can we add a "Projects currently available can be found in Github" line like in your linked PR?

Reply:
Sure :+1:

Reply:
Do we need govuk as a stackname option too?

Reply:
We do, good spot.